### PR TITLE
MEN-7144 - configure SSO on user creation 

### PR DESCRIPTION
**Pull request by mzedel**
Not possible to add if a password related input has input:
Screenshot 2024-05-24 at 01 43 08

Screenshot 2024-05-24 at 15 34 43
Screenshot 2024-05-24 at 15 35 01

### CODE REVIEW

**Review by mineralsfree — APPROVED** [review]
Legitimate

**Comment by mzedel** [comment]
@mineralsfree changed this slightly to only disable the checkbox and not hide it after chatting with @michaelntech - could you take another look please?

File: src/js/components/common/forms/formcheckbox.js [hunk 1]
```diff
@@ -25,7 +25,9 @@ export const FormCheckbox = ({ className, control, id, handleClick, style, label
       return (
         <FormControlLabel
           className={className}
-          control={<Checkbox name={id} onClick={handleClick} checked={value} style={style} color="primary" onChange={() => onChange(!value)} />}
+          control={
+            <Checkbox {...checkboxProps} name={id} onClick={handleClick} checked={value} style={style} color="primary" onChange={() => onChange(!value)} />
```
Comment (mineralsfree) [inline src/js/components/common/forms/formcheckbox.js:29]: I would avoid props spreading as it damages readability (taking into account we don't use typescript)

Comment (mzedel) [reply src/js/components/common/forms/formcheckbox.js]: The spread aligns with [upstream/ mui](https://github.com/mui/material-ui/blob/master/packages/mui-material/src/Checkbox/Checkbox.js) though 😉 - should now be just the `disabled` prop 👌

**Pull request merged**